lib/src/components/NxDropdownMenu/types.tsx

```diff
@@ -4,9 +4,16 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { ReactNode } from 'react';
+import { HTMLAttributes } from 'react';
+import * as PropTypes from 'prop-types';
 
-export interface Props {
+export interface Props extends Omit<HTMLAttributes<HTMLDivElement>, 'className'> {
   onClosing: () => void;
-  children?: ReactNode | null;
+  className?: string | null;
 }
+
+export const propTypes: PropTypes.ValidationMap<Props> = {
+  onClosing: PropTypes.func.isRequired,
+  className: PropTypes.string,
+  children: PropTypes.node
+};
```

Comment:
Is there a reason why we're omitting `className` and then redefining the type?

Reply:
> Is there a reason why we're omitting className and then redefining the type?

Hmm, when I made these changes I got some sort of type error without that redefinition.  But I don't remember what it was, and I can't reproduce it now.  So I've tentatively undone that. 2bb95d2f